boards/raspberrypi/rpi_pico2/board.yml

```diff
@@ -4,3 +4,6 @@ board:
   vendor: raspberrypi
   socs:
   - name: rp2350a
+    variants:
+    - name: w
+      cpucluster: m33
```

Comment:
this should be a different board (or board revision), it is not a variant

Reply:
This is modified from file  boards/raspberrypi/rpi_pico/board.yml, only the cpucluster field is added.